packages/app/src/templates/reducers/template.reducer.test.ts

```diff
@@ -176,4 +176,30 @@ describe("Delete Template data functionality", () => {
       deletingState: TemplateDeletingState.Success,
     })
   })
+
```

Comment:
No tests for `TemplatesEvent.HideDeleteModal`?

Reply:
Test added in [7c222ee](https://github.com/mudita/mudita-center/pull/764/commits/7c222eed5a12a682f3d9b7d9ec91c06d8a7edc54)